bionic/flow.py

```diff
@@ -1611,6 +1627,12 @@ def __init__(self):
 
         self.setting = SettingMethod()
 
+    def _set_state(self, state, uuid):
```

Comment:
Since this does more than just set the `_state` attribute, maybe something like `_initialize` would be more accurate?